### PR TITLE
remove label-changed event filter for daemon node controller

### PR DESCRIPTION
**Pull request by hhyasdf**
Pull Request Description
---

### Does this pull request fix one issue?

Node annotation changed should trigger the reconcile of daemon Node controller.

### Special notes for reviews

### CODE REVIEW

File: pkg/daemon/controller/node.go [hunk 1]
```diff
@@ -304,7 +304,6 @@ func (r *nodeReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	if err := nodeController.Watch(&source.Kind{Type: &corev1.Node{}},
 		&fixedKeyHandler{key: ActionReconcileNode},
 		&predicate.ResourceVersionChangedPredicate{},
-		&predicate.LabelChangedPredicate{},
```
Comment (mars1024) [inline pkg/daemon/controller/node.go:307]: if all we needed is in annotations, how about replacing LabelChangedPredicate with AnnotationChangePredicate?

Comment (hhyasdf) [reply pkg/daemon/controller/node.go:307]: There already has one, https://github.com/alibaba/hybridnet/pull/225/files#diff-bc1d985df6874b40c6a27274b7d19089f4624ce9d94d481cb60f44d7be9047b2L309.

Comment (hhyasdf) [reply pkg/daemon/controller/node.go:307]: Fixed

**Review by mars1024 — APPROVED** [review]
/lgtm

**Pull request merged**